apps_validation/portals.py

```diff
@@ -0,0 +1,51 @@
+from jsonschema import validate as json_schema_validate, ValidationError as JsonValidationError
+
+from apps_exceptions import ValidationErrors
+
+
+IX_NOTES_KEY = 'x-notes'
+IX_PORTAL_KEY = 'x-portals'
+VALIDATION_SCHEMA = {
+    '$schema': 'https://json-schema.org/draft/2020-12/schema',
+    'type': 'object',
+    'properties': {
+        'x-portals': {
+            'type': 'array',
+            'items': {
+                'type': 'object',
+                'properties': {
+                    'name': {
+                        'type': 'string',
+                        'minLength': 1,
+                    },
+                    'path': {
+                        'type': 'string',
+                        'pattern': '^/.*'
+                    },
+                    'scheme': {
+                        'type': 'string',
+                        'enum': ['http', 'https'],
```

Comment:
Wondering if `ftp` or other similar schemes would work, Assuming there is a handler registered on user's browser/system.
We can investigate later.